set.mm

```diff
@@ -500881,6 +501256,228 @@ The sumset operation can be used for both group (additive) operations and
+    qsdrngi.2 $e |- ( ph -> M e. ( MaxIdeal ` O ) ) $.
+    ${
+      qsdrngilem.1 $e |- ( ph -> X e. ( Base ` R ) ) $.
+      qsdrngilem.2 $e |- ( ph -> -. X e. M ) $.
+      $( Lemma for ~ qsdrngi .  (Contributed by Thierry Arnoux, 9-Mar-2025.) $)
+      qsdrngilem $p |- ( ph -> E. v e. ( Base ` Q )
+                           ( v ( .r ` Q ) [ X ] ( R ~QG M ) ) = ( 1r ` Q ) ) $=
+        ( cfv co wceq wcel eqid ad3antrrr vr vm cur cv cmulr cplusg cqg cec cbs
+        wrex wa cqs simpllr ovex ecelqsi syl cvv cnzr cqus ovexd qusbas eleqtrd
+        a1i wb oveq1 eqeq1d adantl crg nzrring c2idl clidl cin mxidlidl syl2anc
+        opprring elind 2idlval eleqtrrdi qusmul2 wer cnsg csubg lidlnsg nsgsubg
+        cmxidl eqger 3syl wss cminusg lidlss ringcld ringidcl simpr oveq2d cgrp
+        wbr c0g ringgrpd oveq1d grpinvcld simplr sseldd grpassd grplidd 3eqtr3d
+        grplinvd eqtrd eqeltrd w3a eqgval biimpar syl23anc qus1 simprd rspcedvd
+        erthi csn cun crsp snssd unssd rspcl rspssid unssad unssbd snssg eldifd
+        mxidlmaxv eleqtrrd elrspunsn mpbid r19.29vva ) ADUCOZUAUDZGDUEOZPZUBUDZ
+        DUFOZPZQZBUDZGDEUGPZUHZCUEOZPZCUCOZQZBCUIOZUJUAUBDUIOZEAYNUUIRZUKZYQERZ
+        UKZYTUKZUUGYNUUBUHZUUCUUDPZUUFQZBUUOUUHUUNUUOUUIUUBULZUUHUUNUUJUUOUURRA
+        UUJUULYTUMZUUIYNUUBDEUGUNUOUPAUURUUHQUUJUULYTAUUBDCUUIUQURCDUUBUSPQAIVC
+        UUIUUIQAUUISZVCADEUGUTJVATVBUUAUUOQZUUGUUQVDUUNUVAUUEUUPUUFUUAUUOUUCUUD
+        VEVFVGUUNUUPYMUUBUHZUUFUUNUUPYPUUBUHUVBUUNUUICDYOUUDEYNGIUUTYOSZUUDSADV
+        HRZUUJUULYTADURRUVDJDVIUPZTZAEDVJOZRZUUJUULYTAEDVKOZFVKOZVLUVGAUVIUVJEA
+        UVDEDWEOREUVIRZUVEKUUIDEUUTVMVNZAFVHRZEFWEOREUVJRAUVDUVMUVEDFHVOUPLFUIO
+        ZFEUVNSVMVNVPDUVGUVIUVJFUVISZHUVJSUVGSZVQVRTZUUSAGUUIRZUUJUULYTMTZVSUUN
+        YPYMUUBUUIAUUIUUBVTZUUJUULYTAEDWAORZEDWBORUVTAUVDUVKUWAUVEUVLDEWCVNEDWD
+        UUBDUUIEUUTUUBSZWFWGTUUNUVDEUUIWHZYPUUIRZYMUUIRZYPDWIOZOZYMYRPZERZYPYMU
+        UBWPZUVFAUWCUUJUULYTAUVKUWCUVLUUIEUVIDUUTUVOWJUPZTZUUNUUIDYOYNGUUTUVCUV
+        FUUSUVSWKZAUWEUUJUULYTAUVDUWEUVEUUIDYMUUTYMSZWLUPZTUUNUWHYQEUUNUWHUWGYS
+        YRPZYQUUNYMYSUWGYRUUMYTWMWNUUNUWGYPYRPZYQYRPDWQOZYQYRPUWPYQUUNUWQUWRYQY
+        RUUNUUIYRDUWFYPUWRUUTYRSZUWRSZUWFSZADWORUUJUULYTADUVEWRTZUWMXFWSUUNUUIY
+        RDUWGYPYQUUTUWSUXBUUNUUIDUWFYPUUTUXAUXBUWMWTUWMUUNEUUIYQUWLUUKUULYTXAZX
+        BZXCUUNUUIYRDYQUWRUUTUWSUWTUXBUXDXDXEXGUXCXHUVDUWCUKUWJUWDUWEUWIXIYPYMY
+        RUUBEDUWFVHUUIUUTUXAUWSUWBXJXKXLXPXGUUNUVDUVHUVBUUFQZUVFUVQUVDUVHUKCVHR
+        UXEDECYMUVGIUVPUWNXMXNVNXGXOAYMEGXQZXRZDXSOZOZRYTUBEUJUAUUIUJAYMUUIUXIU
+        WOAUUIDUXIEGUUTUVEKAUVDUXGUUIWHZUXIUVIRUVEAEUXFUUIUWKAGUUIMXTYAZUUIDUVI
+        UXGUXHUXHSZUUTUVOYBVNAEUXFUXIAUVDUXJUXGUXIWHUVEUXKUUIDUXGUXHUXLUUTYCVNZ
+        YDAGUXIEAUVRUXFUXIWHZGUXIRZMAEUXFUXIUXMYEUVRUXOUXNGUXIUUIYFXKVNNYGYHYIA
+        YMUUIYRDYOUBEUXHGUWRUAUXLUUTUWTUVCUVEUWSUVLAGUUIEMNYGYJYKYL $.
+    $}
+
+    $( A quotient by a maximal ideal is an division ring.  (Contributed by
```

Comment:
```suggestion
    $( A quotient by a maximal left and maximal right ideal is a division ring.  (Contributed by
```
A maximal ideal need not be maximal among all proper left ideals.

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4702/commits/2b407945f95fcbc3bb20aefeeb27272ffc70c821

Reply:
Thanks. I won't have time to review the rest of the MR, so I removed myself from reviewers.

Reply:
OK, thanks for your time anyway!